Source/WebKit/UIProcess/API/Cocoa/NSAttributedString.mm

```diff
@@ -138,9 +144,13 @@ @implementation _WKAttributedStringWebViewCache
     return configuration;
 }
 
-// FIXME (264780): This should ideally default to `NO`, but making this change would break
-// copy/paste from Chrome or Firefox on macOS into TextEdit and other native apps.
-static BOOL shouldAllowNetworkLoads = YES;
+static RetainPtr<NSString>& sourceApplicationBundleIdentifier()
+{
+    static NeverDestroyed<RetainPtr<NSString>> identifier;
```

Comment:
Should this be explicitly initialized to `nil`?

Reply:
The `RetainPtr` constructor will default to `nil`, but I can explicitly initialize it for clarity.

Thanks for the review!

Reply:
> but I can explicitly initialize it for clarity.

Please don't :)

Reply:
The fact that smart pointers are null-initialized by default should not require clarification.

Reply:
....sounds good, will leave it as-is 😅